pytest.ini

```diff
@@ -11,3 +11,7 @@ addopts =
     --durations 10
     --durations-min 1
     --strict-markers
+    --cov
```

Comment:
@samdoran I also recommend adding an explicit `-p pytest_cov` to make sure that this plugin is pre-loaded earlier (this is sometimes important) rather than later (on the plugin discovery stage).

Reply:
Another comment is that you should measure the coverage for both the project and the tests. For this, you'd need to use a few subsequent args: `--cov=ansible_runner --cov=test/`. It may be really helpful to find out if some of the tests never run in CI, for example.

Here's what the coveragepy author posted last year: https://nedbatchelder.com/blog/202008/you_should_include_your_tests_in_coverage.html.